api/locations/index.ts

```diff
@@ -21,46 +22,61 @@ const url: string = getConfigValue(SERVICES_PRD_API_URL);
  */
 export async function getLocations(req: EnhancedRequest, res: Response, next: NextFunction) {
   const searchTerm = req.query.searchTerm;
-  const serviceIds = req.query.serviceIds;
+  // const serviceIds = req.query.serviceIds;
   const locationType = req.query.locationType;
-  const serviceIdArray = serviceIds.split(',');
-  const courtTypeIds = getCourtTypeIdsByService(serviceIdArray);
+  /* const serviceIdArray = serviceIds.split(',');
+  const courtTypeIds = getCourtTypeIdsByService(serviceIdArray); */
   // tslint:disable-next-line:max-line-length
-  const markupPath: string = `${url}/refdata/location/court-venues/venue-search?search-string=${searchTerm}&court-type-id=${courtTypeIds}`;
+  // const markupPath: string = `${url}/refdata/location/court-venues/venue-search?search-string=${searchTerm}&court-type-id=${courtTypeIds}`;
+  const markupPath: string = `${url}/refdata/location/court-venues/venue-search?search-string=${searchTerm}`;
   try {
-    const { status, data }: { status: number, data: LocationModel[] } = await handleGet(markupPath, req);
-    let result: LocationModel[] = data;
+    const headers = setHeaders(req);
+    const response: AxiosResponse<any> = await http.get(markupPath, { headers });
+    let result: LocationModel[] = response.data;
     if (locationType === LocationTypeEnum.HEARING) {
-      result = data.filter(location => location.is_hearing_location === 'Y');
+      result = result.filter(location => location.is_hearing_location === 'Y');
     } else if (locationType === LocationTypeEnum.CASE_MANAGEMENT) {
-      result = data.filter(location => location.is_case_management_location === 'Y');
+      result = result.filter(location => location.is_case_management_location === 'Y');
     }
-    res.status(status).send(result);
+    res.status(response.status).send(result);
   } catch (error) {
     next(error);
   }
 }
 
+/**
+ * Get locations
+ *
+ */
 export async function getLocationsById(req: EnhancedRequest, res: Response, next: NextFunction) {
-  const ids = req.query.ids;
-  const path: string = `${url}/refdata/location/court-locations?epimms_id={}`;
+  const id = req.query.ids;
   try {
-    const { status, data } = await handleGet(path.replace('{}', ids), req);
-    res.status(status).send(data);
+    const basePath = getConfigValue(SERVICES_LOCATION_API_PATH);
+    const path: string = prepareGetLocationsUrl(basePath);
+    const response = await handleLocationGet(path, req);
+    const filteredResults = response.data.court_venues.filter(courtVenue =>
+      courtVenue.epimms_id === id
+    );
+    const mappedLocationModel = mapCourtVenuesToLocationModels(filteredResults);
+    res.send(mappedLocationModel).status(response.status);
   } catch (error) {
     next(error);
   }
 }
 
-function getCourtTypeIdsByService(serviceIdArray: string[]): string {
```

Comment:
Can we not just remove these functions if there are not being used?

Reply:
I'll check if they are and remove if not